### PR TITLE
chore: set $nation budget for each guild in coordinape (#1)

### LINKED ISSUE #1: Convert Coordinape payout CSVs to Gnosis format
https://app.dework.xyz/nation3/rewards-management?taskId=1ed92f8b-d2aa-4dac-a383-c98f1983fe5d

### PR DESCRIPTION
**Pull request by aahna-ashina**
# 🍌🦍

Set the amount of $NATION budgeted for each guild.

The code used for calculating the payouts is in [index.ts](https://github.com/nation3/coordinape/blob/main/index.ts).

close #1

### CODE REVIEW

**Comment by aahna-ashina** [comment]
@RayomndC Please review the generated CSVs: https://github.com/nation3/coordinape/pull/2/files

File: GuildBudget.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-    RewardsGuild = 0.10,
+    EventsGuild = 0.00,
+    RewardsGuild = 0.20,
     ResearchGuild = 0.10,
-    DevelopersGuild = 0.10,
-    MetaGuild = 0.10,
-    GrowthGuild = 0.10
+    DevelopersGuild = 0.25,
+    MetaGuild = 0.05,
+    GrowthGuild = 0.05
```
[the remark this answers is not in the document]
Comment (aahna-ashina) [reply GuildBudget.ts:12]: > Javascript floating point precision will lead to small errors here, I would use percentage values and then in the calculations add a `/100` for better accuracy.

@johnmark13 These are weekly `$NATION` amounts (not percentages).  So `GrowthGuild = 0.10` means a budget of `0.10 $NATION` per week.  Do you think the enum/properties should be renamed to clarify this?

File: index.ts [hunk 2]
```diff
@@ -75,19 +92,10 @@ convertCoordinapePayouts()
...
     // Set column names
     const writer = csvWriter.createObjectCsvWriter({
         path: filePathDisperse,
-        header: ['address', 'received']
-    })
-
-    // Set $NATION amount to match the percentage of total Circle tokens allocated
-    csvRows.forEach(function(row) {
-        row.received = (row.received / totalCircleTokensAllocated) * guildBudget
+        header: ['receiver', 'amount']
```
[the remark this answers is not in the document]
Comment (aahna-ashina) [reply index.ts]: @johnmark13 Renamed the columns: https://github.com/nation3/coordinape/pull/2/commits/006d7835dbe24ea9d74fc60de964184cd2414841

**Comment by xPi2** [comment]
In regards of the code part, I think my only comment is that probably the GuildBudget enum would be better a config file or something similar more input-able.

In regards of the actual weights (budget per guild), I would go through an official proposal to the DAO. I think it would be good to clarify to the DAO what work is expected from each guild each month.

**Review by xPi2 — APPROVED** [review]
In regards of the code part, I think my only comment is that probably the GuildBudget enum would be better a config file or something similar more input-able.

In regards of the actual weights (budget per guild), I would go through an official proposal to the DAO. I think it would be good to clarify to the DAO what work is expected from each guild each month.

**Comment by xPi2** [comment]
@aahna-ashina I think with Parcel we don't need this any time soon. Correct me if I'm wrong.

**Comment by aahna-ashina** [comment]
> @aahna-ashina I think with Parcel we don't need this any time soon. Correct me if I'm wrong.

@xPi2 Yes, with the switch to Parcel, this code becomes obsolete :-D

**Pull request closed without merging**